v4l2.c

```diff
@@ -597,6 +601,9 @@ bool video_engine_capabilities_test(int video_fd,
 	unsigned int capabilities;
 	int rc;
 
+	if(!quiet)
```

Comment:
This and the changes below feel overkill.

Reply:
keep it in my brach. Just for testing ....